components/ca-form.js

```diff
@@ -333,8 +333,8 @@ define([
 
                 // erase current form
                 clearElement(this.form);
-                this.setAttribute('create-url', '');
-                this.setAttribute('update-url', '');
+                this.removeAttribute('create-url');
```

Comment:
Because it's "normal" to reflect content attributes onto properties (IDL attributes) for HTML elements. See http://w3c.github.io/html/infrastructure.html#reflecting-content-attributes-in-idl-attributes 

As for removing an attribute, that's not clear from the above spec for string attributes, but it makes CSS styling around the presence or absence of particular things easier if attributes are removed rather than empty when not set.